packages/babel-plugin-transform-runtime/src/definitions.js

```diff
@@ -5,7 +5,11 @@ module.exports = {
     Map: "map",
     WeakMap: "weak-map",
     Set: "set",
-    WeakSet: "weak-set"
+    WeakSet: "weak-set",
+    Dict: "dict",
+    setImmediate: "set-immediate",
+    clearImmediate: "clear-immediate",
+    delay: "delay"
```

Comment:
No ideas what to do in this case with prototype methods transformed for runtime as statics %)